src/main/java/duke/TimeParser.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class TimeParser {
```

Comment:
It's good that you abstracted out the parsing of time into a separate class! Makes your code neater overall :D